src/v2/BaseLoginRouter.ts

```diff
@@ -291,6 +291,8 @@ class BaseLoginRouter extends Router<Settings, BaseLoginRouterOptions> {
     const authClient = this.settings.getAuthClient();
     delete authClient.options['recoveryToken'];
     this.settings.unset('recoveryToken');
+    // clear otp (email magic link), if any
+    this.settings.unset('otp');
```

Comment:
fix for OKTA-495883 - Back to sign in broken for embedded magic link initiated flows